test/demo/utils/jestExtensions.ts

```diff
@@ -9,6 +12,13 @@ declare global {
              * @param message Optionally, assert that a given error message is returned
              */
             toThrowFDC3Error<R>(code: string, message?: string | RegExp): R;
+
+            /**
+             * Used to test that a RemoteChannel represents the expected client-side Channel
+             * @param channel An object with optional `id` and `type` properties that if present we expect the channel to match
+             * @param classType The class we expect the channel to be an instance of
+             */
+            toBeChannel(channel: {id?: ChannelId, type?: string} | string, classType?: Function): R;
```

Comment:
What's the `| string` part for? For matching just the channel ID?..

Reply:
Changed to Channel ID